packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesFileDownloader.m

```diff
@@ -84,50 +84,49 @@ - (void)downloadManifestFromURL:(NSURL *)url
                                                      timeoutInterval:EXUpdatesDefaultTimeoutInterval];
   [self _setManifestHTTPHeaderFields:request];
   [self _downloadDataWithRequest:request successBlock:^(NSData *data, NSURLResponse *response) {
+      
+    NSHTTPURLResponse* httpResponse = (NSHTTPURLResponse *)response;
+    NSDictionary* headerDictionary = [httpResponse allHeaderFields];
+    NSString* headerSignature = [headerDictionary valueForKey:@"expo-manifest-signature"];
```

Comment:
```suggestion
    id headerSignature = headerDictionary[@"expo-manifest-signature"];
```
we try to stick with this modern syntax for accessing dictionary values even though it's functionally equivalent to calling the selector